python/datasources/cdc_wisqars_youth.py

```diff
@@ -0,0 +1,195 @@
+import pandas as pd
+
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util, standardized_columns as std_col
+from ingestion.cdc_wisqars_utils import (
+    convert_columns_to_numeric,
+    standardize_and_merge_race_ethnicity,
+    WISQARS_COLS,
+)
+from ingestion.constants import (
+    CURRENT,
+    HISTORICAL,
+    NATIONAL_LEVEL,
+    US_NAME,
+)
+from ingestion.dataset_utils import (
+    generate_pct_rel_inequity_col,
+    generate_pct_share_col_with_unknowns,
+    preserve_only_current_time_period_rows,
+)
+from ingestion.merge_utils import merge_state_ids
+
+
+DATA_DIR = "cdc_wisqars"
+
+"""
+Data Source: CDC WISQARS Youth (data on gun violence)
+
+Description:
+- The data on gun violence by youth and race is downloaded from the CDC WISQARS database.
+- The downloaded data is stored locally in our directory for subsequent use.
+
+Instructions for Downloading Data:
+1. Visit the WISQARS website: https://wisqars.cdc.gov/reports/
+2. Select the desired injury outcome:
+   - select fatal data
+3. Choose the demographic selections for the report:
+   - select the appropriate data years, geographic level, and other demographic groups, i.e (age, race)
+4. Select the mechanism (Firearm)
+5. Select appropriate report layout:
+   - For fatal: Year, Intent, State(only if state-level data is needed), and demographic (if not all)
+
+Notes:
+- There is no county-level data
+- Race data is only available for fatal data and is available from 2018-2021
+
+Last Updated: 2/24
+"""
```

Comment:
this is great! I would say let's even remove the choices you list here, and have these instructions reflect exactly the choices that were last made to generate the tables stores in /data. maybe update line 31 with the specific `/data` location as well; imagine this note is for a dev who is brand new and you and i are no longer on the team